homeassistant/components/openuv/config_flow.py

```diff
@@ -10,42 +10,30 @@
     CONF_LATITUDE,
     CONF_LONGITUDE,
 )
-from homeassistant.core import callback
 from homeassistant.helpers import aiohttp_client, config_validation as cv
 
-from .const import DOMAIN
+from .const import DOMAIN  # pylint: disable=unused-import
 
-
-@callback
-def configured_instances(hass):
-    """Return a set of configured OpenUV instances."""
-    return {
-        f"{entry.data.get(CONF_LATITUDE, hass.config.latitude)}, "
-        f"{entry.data.get(CONF_LONGITUDE, hass.config.longitude)}"
-        for entry in hass.config_entries.async_entries(DOMAIN)
+CONFIG_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_API_KEY): str,
+        vol.Optional(CONF_LATITUDE): cv.latitude,
+        vol.Optional(CONF_LONGITUDE): cv.longitude,
```

Comment:
Probably good to mark it as inclusive, so if latitude is given, require longitude too.